netbox/dcim/models/devices.py

```diff
@@ -1175,3 +1175,25 @@ def primary_ip(self):
             return self.primary_ip4
         else:
             return None
+
+    def clean(self):
+        super().clean()
+        vc_interfaces = self.device.vc_interfaces(if_master=False)
```

Comment:
Shouldn't we limit this to interfaces assigned to the VDC?

Reply:
Yes, I have went ahead and made this change.